### ISSUE TITLE
Google Analytics integration issue IncompatibleClassChangeError

### ISSUE THREAD

**Issue by aminelaadhari** [segmentio/analytics-android#423]
Is it the good place to report integrations issues ?

we have this crash for some Android 5 phones:

Fatal Exception: java.lang.IncompatibleClassChangeError: Class 'java.lang.String' does not implement interface 'java.util.Map' in call to 'java.util.Set java.util.Map.entrySet()' (declaration of 'java.lang.reflect.ArtMethod' appears in /system/framework/core-libart.jar)
       at com.google.android.gms.analytics.Tracker.zzb(Unknown Source)
       at com.google.android.gms.analytics.Tracker.send(Unknown Source)
       at com.google.android.gms.analytics.Tracker.set(Unknown Source)
       at com.google.android.gms.analytics.Tracker.setAnonymizeIp(Unknown Source)
       at com.segment.analytics.android.integrations.google.analytics.DefaultTracker.setAnonymizeIp(DefaultTracker.java:27)
       at com.segment.analytics.android.integrations.google.analytics.GoogleAnalyticsIntegration.(GoogleAnalyticsIntegration.java:97)
       at com.segment.analytics.android.integrations.google.analytics.GoogleAnalyticsIntegration$1.create(GoogleAnalyticsIntegration.java:57)
       at com.segment.analytics.Analytics.performInitializeIntegrations(Analytics.java:1009)
       at com.segment.analytics.Analytics$2$1.run(Analytics.java:215)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:135)
       at android.app.ActivityThread.main(ActivityThread.java:5254)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:977)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:763)

**Comment by f2prateek** [comment]
It's best to report these on the specific integration repos https://github.com/segment-integrations/analytics-android-integration-google-analytics.

Which version of  the google analytics sdk are your using?

**Comment by aminelaadhari** [comment]
I looked for the repo but I didn't find it. Do you want me to close this one and open it again there ?

It's v1.0.0

**Comment by f2prateek** [comment]
This is ok for now, but just for the future (they're all under a different org https://github.com/segment-integrations).

I meant the Google Analytics SDK (not the integration). Can you share the output of `grade dependencies`?

**Comment by aminelaadhari** [comment]
+--- com.segment.analytics.android.integrations:google-analytics:+ -> 1.0.0
|    \--- com.google.android.gms:play-services-analytics:8.3.0 -> 8.4.0 (*)

**Comment by f2prateek** [comment]
` 8.4.0 (*)`

That's the issue - your app is using 8.4.0 of the Play Services SDK which breaks binary compatibility with 8.3.0 of the Play Services SDK (which is what our integration uses).

**Comment by f2prateek** [comment]
The solution is to either force your app to use 8.3.0 or wait for us to update the integration to use 8.4.0. If nothing breaks then we'll release [this PR](https://github.com/segment-integrations/analytics-android-integration-google-analytics/pull/1/files) soon

**Comment by f2prateek** [comment]
Released 1.1.0 of the GA integration. Please reopen the issue in the GA repo if the issue persists.

**Comment by tsaisean** [comment]
The issue still exist with 'com.segment.analytics.android.integrations:google-analytics:1.1.3'.

```
Fatal Exception: java.lang.IncompatibleClassChangeError: Class 'java.lang.String' does not implement interface 'java.util.Map' in call to 'java.util.Set java.util.Map.entrySet()' (declaration of 'java.lang.reflect.ArtMethod' appears in /system/framework/core-libart.jar)
       at com.google.android.gms.analytics.Tracker.zzb(Unknown Source)
       at com.google.android.gms.analytics.Tracker.send(Unknown Source)
       at com.google.android.gms.analytics.Tracker.set(Unknown Source)
       at com.google.android.gms.analytics.Tracker.setAnonymizeIp(Unknown Source)
       at com.segment.analytics.android.integrations.google.analytics.DefaultTracker.setAnonymizeIp(SourceFile:27)
       at com.segment.analytics.android.integrations.google.analytics.GoogleAnalyticsIntegration.(SourceFile)
       at com.segment.analytics.android.integrations.google.analytics.GoogleAnalyticsIntegration$1.create(SourceFile:59)
       at com.segment.analytics.Analytics.performInitializeIntegrations(SourceFile:1275)
       at com.segment.analytics.Analytics$2$1.run(SourceFile:238)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:135)
       at android.app.ActivityThread.main(ActivityThread.java:5345)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:947)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:742)
```
